stools/clair/__init__.py

```diff
@@ -17,75 +17,125 @@
 You should have received a copy of the GNU Affero General Public License
 along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
-'''
+"""
 
-from stools.version import __version__
-from stools.clair.image import export_to_targz
-from stools.clair.api import Clair
-from stools.clair.server import start
-from multiprocessing import Process
 import argparse
-import os
-import requests
+import json
 import shutil
 import sys
-import tempfile
-import time
+from multiprocessing import Process
+from os import rename
+from os.path import isdir, join, basename, splitext
+from time import sleep
 
+import requests
+from json2html import *
+from lxml import etree, html
 
-def get_parser():
-    parser = argparse.ArgumentParser(description="Singularity Clair Scanner")
+from stools.clair.api import Clair
+from stools.clair.image import export_to_targz
+from stools.clair.server import start
+from stools.version import __version__
 
-    parser.add_argument('--version', dest="version", 
-                        help="show version and exit.", 
-                        default=False, action='store_true')
 
-    parser.add_argument('--start-server', dest="server", 
-                        help="If running natively, start the web server too.", 
-                        default=True, action='store_true')
+def get_parser():
+    parser = argparse.ArgumentParser(prog="Singularity Clair Scanner", usage=usage())
+
+    parser.add_argument(
+        "--version",
+        action="version",
+        version=version(),
+        dest="version",
+        help="show version and exit",
+        default=False,
+    )
+
+    parser.add_argument(
+        "--start-server",
+        dest="server",
+        help="if running natively, start the web server too",
+        default=True,
+        action="store_true",
+    )
+
+    parser.add_argument(
+        "images", nargs="*", help="Singularity images to scan", type=str
+    )
+
+    parser.add_argument(
+        "--port",
+        default=8080,
+        help="port to serve application (default 8080)",
+        type=int,
+    )
+
+    parser.add_argument(
+        "--report",
```

Comment:
Why is this a variable number of args? You could remove nargs and default to None.

Reply:
It was so that you could pass just --report and it was have a default path of /code/reports or if a user specified a path it would use that instead. If its default to None then a user has to specify a path regardless, no?

Reply:
Reverted :+1: